sdk/python/pulumi_kubernetes/__init__.py

```diff
@@ -26,6 +26,9 @@
     "scheduling",
     "settings",
     "storage",
+    "helm",
```

Comment:
Drive-by review: Nit: It looks like all the other lines are sorted alphabetically.

Reply:
Yeah, this file is generated from a mustache template, and these packages weren't part of the templated list. It's nontrivial to add them, so I just put them at the end.

Reply:
Ah, right. Sorry for the noise!